groovy/postInit/metallurgy/Superalloys.groovy

```diff
@@ -5,14 +5,14 @@ ABS = recipemap('alloy_blast_smelter')
 
 ABS.recipeBuilder()
     .circuitMeta(6)
-    .inputs(ore('dustIron') * 12)
-    .inputs(ore('dustNickel') * 8)
-    .inputs(ore('dustChrome') * 12)
-    .inputs(ore('dustManganese') * 1)
-    .inputs(ore('dustCopper') * 2)
-    .inputs(ore('dustNiobium') * 1)
-    .inputs(ore('dustTantalum') * 1)
-    .inputs(ore('dustMolybdenum') * 2)
+    .inputs(ore('dustAnyPurityIron') * 12)
```

Comment:
these recipes are removed by #1376. We might need a mixin to make the ABS uses anyPurity dusts.